src/locales/hu.yaml

```diff
@@ -1,508 +1,499 @@
-      mcu_bandwidth: '{mcu} Sávszélesség'
-      mcu_load: '{mcu} Terheltsége'
-      model: CPU Modell
-      moonraker_load: Moonraker Terheltsége
-      processor_desc: Processzor
-      product_name: Termék név
-      serial_number: Sorozatszám
-      system_load: Rendszer Terheltsége
-      system_memory: Rendszermemória
-      system_utilization: Rendszerhasználat
-      total_memory: Teljes memória
-  tool:
-    btn:
-      home_x: X
-      home_y: 'Y'
-    tooltip:
-      extruder_disabled: extruder letiltva, min_extrude_temp (%{min}<small>°C</small>) alatt van
-      home_xy: XY kezdőpont
-      home_z: Z kezdőpont
-  version:
-    btn:
-      check_for_updates: Frissítések keresése
-      finish: Befejezve
-      update: Frissítés
-      view_versions: Verziók megtekintése
-    label:
-      commit_history: Történet rögzítése
-      commits_on: vállalja
-      committed: Elkötelezett
-      dirty: PISZKOS
-      invalid: ÉRVÉNYTELEN
-      os_packages: OS csomagok
-      package_list: Csomaglista
-      up_to_date: NAPRAKÉSZ
-      updates_available: Frissítések állnak rendelkezésre
-    status:
-      finished: A frissítések befejeződtek
-      updating: Frissítés...
-    title: Szoftverfrissítések
-    tooltip:
-      commit_history: Előzmények rögzítése
-      dirty: levált fejet jelez, nem a mesteren vagy érvénytelen eredetet
-      invalid: a repo helyi változásait jelzi
-      packages: Csomagok
-      release_notes: Kiadási megjegyzések
+app:
+  bedmesh:
+    label:
+      box_scale: Skála
+      flat_surface: Lapos sík megjelenítése
+      mesh_matrix: Hálós mátrix
+      probed_matrix: Próbamátrix
+      profile_name: Profil Név
+      remove_profile: %{name} profil eltávolítása
+      scale: Színskála
+      wireframe: Drótkeret
+    msg:
+      hint: >-
+        Ha nem a (z) %{name}fájlként menti, választhatja a (z) %{name} profil
+        eltávolítását is
+      not_found: Nem található meglévő ágyháló.
+      not_loaded: Nincs háló betöltve
+    tooltip:
+      calibrate: Új kalibrálás indítása, "alapértelmezett" profilként mentve
+      delete: Törlöm a profilt. Ezután ÚJRAIDÍTOM a nyomtatót!
+      load: Profil Betöltése
+      save: >-
+        A kalibrált profilt a printer.cfg fájlba írom. Ezután ÚJRAIDÍTOM a
+        nyomtatót!
+  chart:
+    label:
+      current: Aktuális
+      item: Név
+      'off': Táblázat KI
+      'on': Táblázat BE
+      power: Kapacitás
+      target: Cél
+      rate_of_change: Hőváltozás
+    tooltip:
+      help: >-
+        A nagyításhoz tartsd lenyomva a Shift billentyűt.<br />Kattints egy
+        elemre a grafikonon való megjelenítéshez.<br />Kattints az erősségre a
+        grafikonon való megjelenítéshez.
+  console:
+    label:
+      auto_scroll: Automatikus görgetés
+      flip_layout: Fordított elrendezés
+      hide_temp_waits: Hőmérsékletek elrejtése
+    placeholder:
+      command: Az automatikus kiegészítéshez használd a 'tab'-ot, parancslistához

```

Comment:
As this is a multiline translation, this line needs to keep the `>-` and then put the rest of the translation on the next line indented by 2 spaces (as it was before the change)